var/spack/repos/builtin/packages/lwtnn/package.py

```diff
@@ -14,9 +14,10 @@ class Lwtnn(CMakePackage):
 
     maintainers = ['haralmha']
 
+    version('2.13', sha256='d5c1cede00f9d9cc0a25f070f744999453aec477dae6ef02701c11543c5b9553')
     version('2.12.1', sha256='b820e698d4ed60737e646ca87a42354e8ac548403348b7f2940e8fda1c0f8203')
     version('2.10', sha256='bf84b290c44da582226344b0d5febf7fdbd1cbdee94fcc8bcac972c7355564ed')
 
     depends_on('boost@1.54:')
-    depends_on('eigen@:3.3.99', when='@2.11:')
+    depends_on('eigen@:3.3.99', when='@2.11:2.12')
```

Comment:
Can you add a comment to explain why `eigen` has an upper bound? Also (minor), can you invert the `eigen` dependencies so they read:
```python
depends_on('eigen')
depends_on('eigen@:3.3.99', when='@2.11:2.12')
```

Reply:
@alalazo done.